pkgs/development/python-modules/penelope/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchFromGitHub
+}:
+buildPythonPackage rec {
+  name = "penelope";
+  version = "0.9.2";
+
+  src = fetchFromGitHub {
+    owner = "brightio";
+    repo = "penelope";
+    rev = "v${version}";
+    hash = "sha256-KlbB7i009UttTp+/MaF3y9CGUkAGEQQq/W+hQYKk9CY=";
+  };
+
+  preBuild = ''
+    cat > setup.py << EOF
+    from setuptools import setup
+
+    setup(
+      name='penelope',
+      version='${version}',
+      scripts=[
+        'penelope.py',
+      ],
+    )
+    EOF
+  '';
+
+  installPhase = "install -Dm755 $src/penelope.py $out/bin/penelope";
```

Comment:
```suggestion
```

Please consider installing the program as a [script](https://flit.pypa.io/en/stable/pyproject_toml.html#scripts-section)

Reply:
I'm now using a script file. Can't use a project script due to the way the tool is written.